File: benchmarks/http.js
const http = require('http');

const server = http.createServer((req, res) => {
    res.writeHead(200, { 'Content-Type': 'text/plain' });
    res.end('Hello from JS Server!');
});

const PORT = 8083;
server.listen(PORT, () => {
    console.log(`JS server listening on port ${PORT}`);
});

File: benchmarks/http.go
package main

import (
	"fmt"
	"net/http"
)

func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		w.Header().Set("Content-Type", "text/plain")
		fmt.Fprintf(w, "Hello from Go Server!")
	})

	fmt.Println("Go server listening on port 8082")
	if err := http.ListenAndServe(":8082", nil); err != nil {
		panic(err)
	}
}
